boards/shields/x_nucleo_idb05a1/doc/index.rst

```diff
@@ -0,0 +1,96 @@
+.. _x-nucleo-idb05a1:
+
+X-NUCLEO-IDB05A1: Bluetooth Low Energy expansion board based on the SPBTLE-RF module
```

Comment:
I'd (still) reduce the title length to just: ``X-NUCLEO-IDB05A1: Bluetooth Low Energy expansion board`` or maybe even ``X-NUCLEO-IDB05A1: BLE expansion board``